### ISSUE TITLE
Add HTTP status page

### ISSUE THREAD

**Issue by tresf** [qzind/tray#324]
As LetsEncrypt support is becoming more and more popular there's an increased request to have QZ Tray report back the certificate expiration of the SSL cert so that site administrators can better manage print servers.

Furthermore, QZ Tray could serve itself through a status page if we properly handled WebSocket upgrades from HTTP to WS gracefully.

Below is a proof of concept which will return a JSON array with some QZ Tray info.  This could be extended to display an HTML status page and `/json` for a dedicated JSON page, etc.

**Requirements:** 
* At a minimum, anything on the About dialog would be useful, so this may end up being a wrapper for both purposes.
* Must display the SSL certificate information
* Should be easily scalable for other purposes (logs, certificates, etc) for the future

![image](https://user-images.githubusercontent.com/6345473/41310689-3ad2b468-6e50-11e8-98fa-1efb8e2645f3.png)

**Comment by tresf** [comment]
Sample output from the above example:

```json
{
   "company": "QZ Industries, LLC",
   "title": "QZ Tray",
   "url": "https:\/\/qz.io",
   "java": "1.8.0+45",
   "version": "2.0.6",
   "ssl": [
      {
         "expires": "Sat May 01 13:34:19 EDT 2038",
         "subject": "CN=localhost,1.2.840.113549.1.9.1=#160d737570706f727440717a2e696f,OU=QZ Industries\\, LLC,O=QZ Industries\\, LLC,L=Canastota,ST=NY,C=US"
      },
      {
         "expires": "Sat May 01 13:34:24 EDT 2038",
         "subject": "CN=localhost,1.2.840.113549.1.9.1=#160d737570706f727440717a2e696f,OU=QZ Industries\\, LLC,O=QZ Industries\\, LLC,L=Canastota,ST=NY,C=US"
      }
   ]
}
```

**Comment by tresf** [comment]
Initial support added under 14ec02c on `html` branch.

![image](https://user-images.githubusercontent.com/6345473/39945788-b1c535da-5539-11e8-866a-cdf6314cea62.png)

![image](https://user-images.githubusercontent.com/6345473/39945801-c5ff19c6-5539-11e8-8f08-53adebaa0711.png)

**Comment by bberenz** [comment]
I've pulled out the about data to a common class and added some basic formatting of said data on the html page under 348014c377cf7d980b41d62795a1049f1f73e629.
You mention using the same data on the about dialog, but there's some special formatting there, so I'm not sure how to do that cleanly yet.
Feel free to edit further as you see fit.

**Comment by tresf** [comment]
Closed via #334.
